bind_exporter.go

```diff
@@ -77,6 +77,11 @@ var (
 		"Number of RRSets in Cache database.",
 		[]string{"view", "type"}, nil,
 	)
+	resolverCacheStats = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, resolver, "cache_stats"),
+		"Number of Cache Stats in Cache database.",
+		[]string{"view", "stat"}, nil,
```

Comment:
I'm not sure what "Cache Stats" are. Prometheus convention is that the metric name and help should be understandable by the end user without having to know what the underlying code implementation is.

Do you have a link to any upstream documentation about these values?

We typically don't just generically expose random values with `"stat"` labels. This provides for a poor user experience.